stack.yaml

```diff
@@ -3,3 +3,4 @@ packages:
 - '.'
 extra-deps:
 - optparse-applicative-0.17.0.0
+- shellwords-0.1.3.0
```

Comment:
Note that I just added `shellwords` to Stackage, so it'll show up in an LTS soon.